sdk/cosmos/azure-cosmos/azure/cosmos/documents.py

```diff
@@ -302,7 +302,7 @@ class TriggerOperation(object):
 class SSLConfiguration(object):
     """Configurations for SSL connections.
 
-    Please refer to http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification for more detail.
+    Please refer to https://requests.readthedocs.io/en/master/user/advanced/#ssl-cert-verification for more detail.
```

Comment:
Annoyingly safari does not properly navigate to the anchor on a page load (chome does). Not sure there's anything that can be done, though.

Reply:
Per our OOB discussion; Initially thought this was due to a 302 redirect in the official readthedocs link, but that seems to have resolved with safari still being unhappy, so I'm scratching my head.

I've obtained a mac for testing going forward, if I can find any clever ways to handle this I'll push changes back to this link as well.